wiki/ranking-guide.md

```diff
@@ -147,11 +174,13 @@ and you manage to get a better accuracy and that play is now worth **260 pp**, y
 amount of PP.
 
 ### How can I improve and get more PP?
-To improve and get a better rank in the global ranking, there are quite a few tools you can use to find ranked songs that
- fit your skill level. First, it’s always good to go see your own [ScoreSaber](https://scoresaber.com/global) profile to
+To improve your skills and to get a spot in the global rankings, there are quite a few tools you can use to find ranked songs that
```

Comment:
"To improve your skills and to get a spot in the global rankings" -> "To improve your skills and position in the global rankings" ?